pimd/pim_msdp.c

```diff
@@ -1321,8 +1321,12 @@ int pim_msdp_config_write(struct pim_instance *pim, struct vty *vty,
 		}
 
 		for (ALL_LIST_ELEMENTS_RO(mg->mbr_list, mbrnode, mbr)) {
-			pim_inet4_dump("<mbr?>", mbr->mbr_ip, mbr_str,
-				       sizeof(mbr_str));
+			if (mbr->mbr_ip.s_addr != INADDR_ANY)
+				pim_inet4_dump("<mbr?>", mbr->mbr_ip, mbr_str,
```

Comment:
In my opinion, you can use %pI4 directly to print the address instead of calling pim_inet4_dump API

Reply:
Ack. change to %pI4